pkg/controller/dataimportcron-controller.go

```diff
@@ -416,7 +420,7 @@ func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *c
 			}
 		}
 	} else if importSucceeded {
-		if err := r.updateDataImportCronSuccessCondition(ctx, dataImportCron, format, snapshot); err != nil {
+		if err := handler.updateOnSuccess(r, ctx, dataImportCron, imports[0].DataVolumeName); err != nil {
```

Comment:
imports is `dataImportCron.Status.CurrentImports`, so why pass it as arg?

Reply:
not doing that anymore